src/contrib/serializers/Akka.Serialization.Hyperion/HyperionSerializer.cs

```diff
@@ -70,7 +70,8 @@ public HyperionSerializer(ExtendedActorSystem system, HyperionSerializerSettings
                     preserveObjectReferences: settings.PreserveObjectReferences,
                     versionTolerance: settings.VersionTolerance,
                     surrogates: new[] { akkaSurrogate },
-                    knownTypes: provider.GetKnownTypes()));
+                    knownTypes: provider.GetKnownTypes(), 
+                    ignoreISerializable:true));
```

Comment:
old behavior would be to ignore it because ISerializable support in Hyperion was introduced only in latest release.
Maybe you can just release a fixed version for "Akka.Serialization.Hyperion" without ISerialization support and discuss this afterwards ?